charts/publisher/templates/web/deployment.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         - name: app
           image: "{{ .Values.common.image.repository }}:{{ .Values.common.image.tag }}"
           command: ["bundle"]
-          args: ["exec", "puma"]
+          args: ["exec", "unicorn -c ./config/unicorn.rb -p ${PORT}"]
```

Comment:
Oh fun, does it not work with puma? Or did something just go wonky during a rebase or something? (only asking cos we had puma working the other day 😅)

Reply:
Oh yikes - sorry about the breakage! I'll go revert the puma commit.

Reply:
Puma stuff reverted in #14.